borg/archiver.py

```diff
@@ -1720,6 +1720,8 @@ def run(self, args):
         setup_logging(level=args.log_level, is_serve=args.func == self.do_serve)  # do not use loggers before this!
         if args.show_version:
             logger.info('borgbackup version %s' % __version__)
+        if 'output_list' in args and args.output_list:
```

Comment:
Yeah, as the commit message said, it was a quick hack because I only had a few minutes to fix the testcase breaks caused by rebasing.  It looks from the argparse docs that 'vars(args)' is the preferred way to treat an argparse Namespace object like a dict.  I'll update that tonight or tomorrow.

-- 
Ed Blackman